package/Scalar.roc

```diff
@@ -18,36 +18,34 @@ interface Scalar
     ]
 
 ## A [Unicode scalar value](http://www.unicode.org/glossary/#unicode_scalar_value) - that is,
-## any [code point](./CodePoint#CodePoint) except for [high-surrogate](http://www.unicode.org/glossary/#high_surrogate_code_point)
-## and [low-surrogate](http://www.unicode.org/glossary/#low_surrogate_code_point) code points.
```

Comment:
Again, I think it is better to link to the source `www.unicode.org` here so that future contributors can easily find the relevant details in the spec.

Reply:
ok 👍